pushy-donkeys/Bangazon-Site
# image upload works

Fixes for #14

Changes proposed in this pull request:
- Can upload an image (if you want to) when creating a product to sell
- Modified the ProductController to accept the image on creation 
- images are uploaded to the database and then to the wwwroot/products folder

How to test:
- Pull my branch / run program
- Go to create a product and upload an image
- See image with its information on the product index

@PushyDonkey/
